docs/whats-new/index.yml

```diff
@@ -1,75 +1,72 @@
-  - title: "Community"
-    linkLists:
-      - linkListType: whats-new
-        links:
-          - text: Community
-            url: https://dotnet.microsoft.com/platform/community
-  - title: "Related what's new pages"
-    linkLists:
-      - linkListType: whats-new
-        links:
-          - text: ASP.NET Core docs updates
-            url: https://docs.microsoft.com/aspnet/core/whats-new/
-          - text: Xamarin docs updates
-            url: https://docs.microsoft.com/xamarin/whats-new/
-          - text: .NET Core release notes
-            url: https://github.com/dotnet/core/blob/master/release-notes/README.md
-          - text: ASP.NET Core release notes
-            url: https://docs.microsoft.com/aspnet/core/release-notes/aspnetcore-3.1
-          - text: C# compiler (Roslyn) release notes
-            url: https://github.com/dotnet/roslyn/tree/master/docs/ide
-          - text: Visual Studio release notes
-            url: https://docs.microsoft.com/visualstudio/releases/2019/release-notes
-          - text: Visual Studio for Mac release notes
-            url: https://docs.microsoft.com/visualstudio/releasenotes/vs2019-mac-relnotes
-          - text: Visual Studio Code release notes
-            url: https://code.visualstudio.com/updates
+### YamlMime:Landing
+
+title: .NET documentation - what's new?
+summary: Welcome to what's new in .NET docs. Use this page to quickly find the latest changes.
+metadata:
+  title: .NET documentation what's new?
+  description: Learn about new and updated content in .NET docs.
+  ms.date: 2020-04-22T00:00:00.0000000
+  ms.topic: landing-page
+landingContent:
+- title: Find .NET updates
+  linkLists:
+  - linkListType: whats-new
+    links:
+    - text: June 2020
+      url: 2020-06.md
+    - text: May 2020
+      url: dotnet-2020-05.md
+    - text: April 2020
+      url: dotnet-2020-04.md
+    - text: March 2020
+      url: dotnet-2020-03.md
+    - text: February 2020
+      url: dotnet-2020-02.md
+    - text: January 2020
+      url: dotnet-2020-01.md
+- title: Get involved - contribute
+  linkLists:
+  - linkListType: overview
+    links:
+    - text: .NET docs repository
+      url: https://github.com/dotnet/docs/blob/master/README.md
+    - text: Project structure and labels for issues and pull requests
+      url: https://docs.microsoft.com/contribute/dotnet/labels-projects
+    - text: .NET Foundation
+      url: https://dotnetfoundation.org/
+  - linkListType: concept
+    links:
+    - text: Microsoft docs contributor guide
+      url: https://docs.microsoft.com/contribute
+    - text: .NET docs contributor guide
+      url: https://docs.microsoft.com/contribute/dotnet/dotnet-contribute
+    - text: .NET API reference docs contributor guide
+      url: https://github.com/dotnet/dotnet-api-docs/blob/master/CONTRIBUTING.md
+    - text: .NET samples contributor guide
+      url: https://github.com/dotnet/samples/blob/master/CONTRIBUTING.md

```

Comment:
This is unrelated to the PR.
But you may consider removing/replacing these links.
Both links redirects the reader to the one in this repo.
I'd only include the one in this repo.